examples/model_selection/plot_grid_search_stats.py

```diff
@@ -0,0 +1,548 @@
+    return t_stat, p_val
+
+
+model_1_scores = model_scores.iloc[0].values  # scores of the best model
+model_2_scores = model_scores.iloc[1].values  # scores of the second-best model
+
+# %%
+differences = model_1_scores - model_2_scores
+
+n = differences.shape[0]  # number of test sets
+df = n - 1
+n_train = len(list(cv.split(X, y))[0][0])
+n_test = len(list(cv.split(X, y))[0][1])
+
+t_stat, p_val = compute_corrected_ttest(differences, n, df, n_train, n_test)
+print(f"Corrected t-value: {t_stat:.3f}\n"
+      f"Corrected p-value: {p_val:.3f}")
+
+# %%
+# We can compare the corrected t- and p-values with the uncorrected ones:
+
+t_stat_uncorrected = (
+    np.mean(differences) / np.sqrt(np.var(differences, ddof=1) / n)
+)
+p_val_uncorrected = t.sf(np.abs(t_stat_uncorrected), df)
+
+print(f"Uncorrected t-value: {t_stat_uncorrected:.3f}\n"
+      f"Uncorrected p-value: {p_val_uncorrected:.3f}")
+
+# %%
+# Using the conventional significance alpha level at `p=0.05`, we observe that
+# the uncorrected t-test concludes that the first model is significantly better
+# than the second.
+#
+# With the corrected approach, in contrast, we fail to detect this difference.
+#
+# In the latter case, however, the frequentist approach does not let us
+# conclude that the first and second model have an equivalent performance. If
+# we wanted to make this assertion we need to use a Bayesian approach.
+
+# %%
+# Comparing two models: Bayesian approach
+# ---------------------------------------
+# We can use Bayesian estimation to calculate the probability that the first
+# model is better than the second. Bayesian estimation will output a
+# distribution that specifies the probability of each parameter value, which in
+# this example is the mean of the differences in the performance of two models.
+#
+# To obtain the posterior distribution we need to define a prior that models
+# our beliefs of how the means are distributed before looking at the data,
+# and multiply it by a likelihood function that computes how likely our
+# observed mean differences are, given the values that the mean of differences
+# could take.
+#
+# Bayesian estimation can be carried out in many forms to answer our question,
+# but in this example we will implement the approach suggested by Benavoli and
+# collegues [4]_.
+#
+# One way of defining our posterior using a `closed-form expression
+# <https://en.wikipedia.org/wiki/Closed-form_expression>`_ is to select a
+# prior `conjugate <https://en.wikipedia.org/wiki/Conjugate_prior>`_ to the
+# likelihood function. Benavoli and collegues [4]_ show that when comparing the
+# performance of two classifiers we can model the prior as a Normal-Gamma
+# distribution (with both mean and variance unknown)
+# `conjugate <https://en.wikipedia.org/wiki/Conjugate_prior>`_ to a normal
+# likelihood, to thus express the posterior as a normal distribution.
+# Marginalizing out the variance from this normal posterior, we can define the
```

Comment:
From my understanding and after reading about this topic in response to the comment of @glemaitre that pointed out that `Frequentist` should not be capitalized, I think `Normal` should not be capitalized as well. Only names that derive from personal nouns should be. Do you think this is not the case?

Reply:
Your're right. However, I see two reasons for the capital in **N**ormal distribution:

- It is the same as the **G**aussian distribution. So we do as if Normal were a name. Site note: We also say **G**amma distribution, I guess because it is written with a capital greek letter.
- To make a distinction between a normal, i.e. usual, distribution and The **N**ormal distribution.

Just stating my thoughts. I've no strong opinion on that.

Reply:
Those seem valid points as well, but since both options are valid then I would keep the one already used so no additional changes have to be made. Do you agree?